2018/CG-09-04.md

```diff
@@ -44,4 +44,188 @@ Installation is required, see the calendar invite.
 
 ## Meeting Notes
 
-Posted after meeting.
+### Opening, welcome and roll call
+
+#### Opening of the meeting
+
+#### Introduction of attendees
+
+* Adam Klein
+* Alex Crichton
+* Alon Zakai
+* Andreas Rossberg
+* Arun Purushan
+* Ben Smith
+* Ben Titzer
+* Conrad Watt
+* Deepti Gandluri
+* Derek Schuff
+* Heejin Ahn
+* Jacob Gravelle
+* Jay Phelps
+* JF Bastien
+* Keith Miller
+* Limin Zhu
+* Luke Wagner
+* Peter Jensen
+* Richard Winterton
+* Shiv Kushwaha
+* Thomas Lively
+* Wouter Van Oortmerssen
+* Yury Delendik
+
+### Find volunteers for note taking (acting chair to volunteer)
+
+Ben Smith taking notes
+
+### Adoption of the agenda
+
+Derek Seconds
+
+### Proposals and discussions
+
+#### Review of action items from prior meeting.
+
+AI(Thomas Lively): Write up discussion about optional vs required features.
+
+TL: 80% written up, will post today.
```

Comment:
Posted at https://github.com/WebAssembly/spec/issues/870 if you want to add a link to the notes.

Reply:
Thanks, added.